src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1876,7 +1877,7 @@ protected Task<Boolean> createTask() {
             @Override
             protected Boolean call() {
               TabulatorSession session = new TabulatorSession(configPath);
-              return session.convertToCdf();
+              return session.convertToCdf(this::updateProgress);
```

Comment:
Where is this actually used? I don't see any place in the GUI that shows a progress bar for converting to CDF.

Reply:
It's not used, happy to remove though I don't see any downsides leaving it in either

Reply:
No problem, fine to leave as-is. Merge away!